docs/guide.md

```
@@ -235,12 +260,18 @@ backend gets `railyard-herdr-2` rather than stealing the first one's client.
 git clone <url> projects/<name>
 ```
 
-Then add a line to [`../data/projects.md`](../data/projects.md):
+Then add a line to `data/projects.md`, creating the file if it is not there:
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)